apps/mochi-web/components/app/detail/AppDetailPageHeader.tsx

```diff
@@ -24,6 +24,7 @@ export const AppDetailPageHeader = ({ name = '' }: Props) => {
   return (
     <PageHeader
       title={name}
+      className="border-b-[0.5px] border-neutral-400"
```

Comment:
I think the update should be in the `PageHeader` component. Also, shouldn't use `0.5px` value, use `border-b` instead and make sure you use the correct color code for the border to support dark mode